.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+jobs:
+  build:
+    parallelism: 3 # run three instances of this job in parallel
+    docker:
+      - image: circleci/ruby:2.4.2-jessie-node
+        environment:
+          BUNDLE_JOBS: 3
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+    steps:
+      - checkout
+
+      - run:
+          name: Which bundler?
+          command: bundle -v
```

Comment:
is this just debug output?

Reply:
(For the task) I think. Surprisingly, `-v` is version, not verbose for `bundler`. I based this on a simple config circle hosted.